server/src/test/resources/application-test.yml

```diff
@@ -38,6 +38,9 @@ flyway:
 ---
 check-ins:
   web-address: "https://checkins.objectcomputing.com"
+  application:
+    google-api:
+      delegated-user: test@objectcomputing.com
```

Comment:
This was added as the property is defined as coming from an env var in application.yml and this would otherwise fail under test